### ISSUE TITLE
Flake8 throws warning on first use of async in file

### ISSUE THREAD

**Issue by User-r00** [AtomLinter/linter-flake8#559]
I'm working on a script that uses the Asyncio module. Flake8 is reporting an E999 - Syntax error on only the first usage of the async command. 

```
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

"""Braveheart - A discord bot that provides moderation tools and security."""

import os
import logging
from time import sleep
import random
import asyncio
import discord
from discord.ext.commands import Bot
from discord.ext import commands
import constants as C

logger = logging.getLogger('Braveheart')
logger.setLevel(logging.DEBUG)
handler = logging.FileHandler(filename='bravehearttest.log',
                              encoding='utf-8', mode='w')
handler.setFormatter(logging.Formatter('%(asctime)s:%(levelname)s:%(name)s: '
                                       '%(message)s'))
logger.addHandler(handler)

# Discord client setup
B = discord.Client(status='online')
bot_prefix = '$'
B = commands.Bot(command_prefix=bot_prefix)


@B.event
async def on_ready():
    """Handles what happens when bot logs in."""

    print('=====================================')
    print('| Braveheart Online')
    print('| Name: {}'.format(B.user.name))
    print('| ID: {}'.format(B.user.id))
    print('=====================================')

    # Discord presence
    await B.change_presence(game=discord.Game(name='JHJlaW5kckdNUw=='))

    print('Listening...')


@B.event
async def on_member_join(member):
    """Welcome message sent to new users."""
    pass
```

In this bit of code, the E999 is given on the c of the first async. However, the second async does not throw an issue. The code runs without issue and this error does not appear in other linters/editors.

**Comment by Arcanemagus** [comment]
Does the CLI `flake8` show the same results?

**Comment by mateusfccp** [comment]
It's happening also with me.

@Arcanemagus, yes, it happens in CLI.

**Comment by Arcanemagus** [comment]
This is just a wrapper around `flake8`, if you are getting errors like that you should report them to `flake8` itself 😉.
